web/src/message_flags.ts

```diff
@@ -94,7 +96,7 @@ export function unstar_all_messages_in_topic(stream_id, topic) {
         ]),
     };
 
-    channel.get({
+    void channel.get({
         url: "/json/messages",
         data,
         success(data) {
```

Comment:
What should I do with this `data` variable? 

`'data' is of type 'unknown'` so `data.messages` doesn't work

I can't cast it to `{messages: Message[]}` because it wants `data` to be explicitly unknown.

Reply:
The recommended solution is to use `zod` to parse the data into the expected type. `git grep zod` will find you various examples.

Reply:
Is it okay if I type the message as only having an id? The full Message type is very complicated and it seems like `zod` wants things written only in terms of primitive types.

Reply:
That seems OK given that we don't inspect the rest, though probably should be changed if/when we start using zod in `message_fetch.js`.

Reply:
Okay cool, I already updated the PR with that